common/src/main/scala/explore/model/decoders.scala

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.implicits._
+import gem.util.Enumerated
+import io.circe.Decoder
+import io.circe.DecodingFailure
+import io.circe.Encoder
+import io.circe.HCursor
+import io.circe.JsonObject
+import io.circe.generic.semiauto.deriveDecoder
+import io.circe.generic.semiauto.deriveEncoder
+import gsp.math.RightAscension
+import gsp.math.Declination
+import gsp.math.Coordinates
+import gsp.math.Epoch
+import gsp.math.ProperMotion
+import explore.model.enum.ObsStatus
+import java.time.Duration
+
+object decoders {
+
+  implicit def enumDecoder[E: Enumerated]: Decoder[E] =
```

Comment:
This is great. We should consider whether these should go to gsp-core eventually

Reply:
Yes, good idea. We will probably use this from other apps too.